{"uid":"23a6a8f9e03b61a2","name":"SyncManager sync throws error when documents sync fails","historyId":".frontend.packages.internal.catalyst_voices_services.test.src.sync.sync_manager:.frontend.packages.internal.catalyst_voices_services.test.src.sync.sync_manager#SyncManager sync throws error when documents sync fails","time":{"start":1743070253000,"stop":1743070253032,"duration":32},"status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"stepsCount":0,"attachmentStep":false},"afterStages":[],"labels":[{"name":"resultFormat","value":"junit"},{"name":"suite","value":".frontend.packages.internal.catalyst_voices_services.test.src.sync.sync_manager"},{"name":"testClass","value":".frontend.packages.internal.catalyst_voices_services.test.src.sync.sync_manager"},{"name":"package","value":".frontend.packages.internal.catalyst_voices_services.test.src.sync.sync_manager"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":0,"broken":0,"skipped":0,"passed":2,"unknown":0,"total":2},"items":[{"uid":"295a3fb39067b680","reportUrl":"https://input-output-hk.github.io/catalyst-voices/allure-action/fix_registration_no_exit_confirm_dialog_2004/test-report/14102283381_1743066636778/#testresult/295a3fb39067b680","status":"passed","time":{"start":1743065640000,"stop":1743065640047,"duration":47}}]},"tags":[]},"source":"23a6a8f9e03b61a2.json","parameterValues":[]}